projects/live-preview/components/si-live-preview-renderer/si-live-preview-renderer.component.ts

```diff
@@ -301,10 +312,6 @@ export class SiLivePreviewRendererComponent implements OnChanges, OnDestroy {
       private container!: ViewContainerRef;
       private childRouteBackup: Routes | undefined;
 
-      constructor() {
-        this.changeDetector.detach();
```

Comment:
So this line was the reason why change detection was not working correctly when switching to zoneless or all onpush. I am not sure why it's added here @dr-itz May be you know. Also I tested some examples keeping on default change detection and that also seems to work but @dr-itz you might have better idea when they fail.